chapters/0-introduction.tex

```diff
@@ -72,76 +194,128 @@ \section{Doing credible research at scale}
+often at different levels of analysis and different stages in time.
+The tight linkages between documentation, data quality, and policy decisions
+have long been recognized by research entities such as government statistical agencies,\cite{jepdataquality}
+and must now be imported into the practice of researchers who collect original data
+rather than relying on data that comes from, for example, highly-experienced statistical agencies.
+
+A breakdown in any part of this data pipeline
+means that the results that are acquired become unreliable.\cite{mccullough2008economics}
+If that happens, the results cannot be faithfully interpreted
+as being an accurate picture of the intended research design.\sidenote{
+  \url{https://blogs.worldbank.org/impactevaluations/more-replication-economics}}
+Because we almost never have ``laboratory'' settings in this type of research,
+such a failure has a very high cost:
+we will have wasted the investments that were made into knowledge generation,
+with little ability to reproduce or recreate the situation
+where we intended to operate the research project.\cite{camerer2016evaluating}
+Hence accurate and reproducible data management and analysis is a core component
+of the credibility of development research.
+Being able to accurately and reproducibily implement these tasks
+is essential to the success and credibility of any modern research output.
+
+
+\section{Documenting data work with standardized code}
+
+One method of solving this problem in the social context
+is what we refer to as \textbf{process standardization}.
+Process standardization means that there is
+little ambiguity about how something ought to be done,
+and therefore the tools to do it can be set in advance.
+Standard processes help other people understand your work.
+Work should be well-documented in the sense that others can:
+(1) quickly understand what a particular process or output is supposed to be doing;
+(2) evaluate whether or not it does that thing correctly; and
+(3) modify it efficiently either to test alternative hypotheses
+or to adapt into their own work.
+
+Modern quantitative research already relies heavily
+on standardized statistical software tools,
+written with various coding languages, to standardize analytical work.
+Outputs like regression tables and data visualizations
+are creating using code in statistical software for two primary reasons.
+The first is that using a standard command or package ensures that the work is done right,
+and the second is that it ensures the same procedure can be confirmed or checked
+at a later date or using different data.
+Keeping a clear, human-readable record of these code and data structures is critical.
+While is often \textit{possible} to perform nearly all the relevant tasks
+through an interactive user interface or even through software such as Excel,
+this practice is strongly advised against.
+In the context of statistical analysis, this approach is widely accepted.
```

Comment:
widely accepted? increasingly perhaps, with pressure from journals ...

Reply:
maybe this is not clear, it's meant to say that "everyone agrees analysis work should be coded" in contrast to, say, data work. does that make sense?

Reply:
ah, that does make sense. it wasn't clear.

Reply:
752fd32f596b4591e7d6453ee16b5bd1882e512b